components/alibi-detect-server/Dockerfile

```diff
@@ -14,28 +14,38 @@ FROM base as builder
 ARG PYTHON_VERSION
 ARG CONDA_VERSION
 
+# mesa-libGL: this is to avoid "ImportError: libGL.so.1" from opencv
+RUN microdnf update -y && \
+    microdnf install -y \
+      unzip mesa-libGL \
```

Comment:
nit + probably personal preference: either I'd keep them in one line or broke down to one item per line

Reply:
Ok, let me break them down to one item per line